NitroxModel/Discovery/InstallationFinders/ConfigFileGameFinder.cs

```diff
@@ -9,26 +9,36 @@ namespace NitroxModel.Discovery.InstallationFinders
     /// </summary>
     public class ConfigFileGameFinder : IFindGameInstallation
     {
-        private const string FILENAME = "path.txt";
+        public delegate string ConfigResolver();
+
+        private static event ConfigResolver getConfig;
+
+        public static void AddResolver(ConfigResolver resolver)
+        {
+            getConfig += resolver;
+        }
 
         public Optional<string> FindGame(List<string> errors = null)
         {
-            if (!File.Exists(FILENAME))
+            if(getConfig == null)
             {
-                errors?.Add($@"Game installation directory config file is not set. Create a '{FILENAME}' in directory: '{Directory.GetCurrentDirectory()}' with the path to the Subnautica installation directory.");
+                // should only happen outside of the launcher
+                errors?.Add("Game installation config resolver is not set.");
                 return Optional.Empty;
             }
+            string config = getConfig.Invoke();
 
-            string path = File.ReadAllText(FILENAME).Trim();
-            if (string.IsNullOrEmpty(path))
+            if (config == "" || config == "False")
```

Comment:
Instead of checking for false it should check if it is a directory with subnautica exe